pandas/core/indexes/base.py

```diff
@@ -1300,17 +1300,22 @@ def set_names(self, names, level=None, inplace=False):
         Int64Index([1, 2, 3, 4], dtype='int64', name='foo')
         >>> pd.Index([1, 2, 3, 4]).set_names(['foo'])
         Int64Index([1, 2, 3, 4], dtype='int64', name='foo')
+
         >>> idx = pd.MultiIndex.from_tuples([(1, u'one'), (1, u'two'),
-                                            (2, u'one'), (2, u'two')],
-                                            names=['foo', 'bar'])
+        ...                                  (2, u'one'), (2, u'two')],
+        ...                                  names=['foo', 'bar'])
         >>> idx.set_names(['baz', 'quz'])
-        MultiIndex(levels=[[1, 2], [u'one', u'two']],
+        MultiIndex(levels=[[1, 2], ['one', 'two']],
                    labels=[[0, 0, 1, 1], [0, 1, 0, 1]],
-                   names=[u'baz', u'quz'])
+                   names=['baz', 'quz'])
         >>> idx.set_names('baz', level=0)
-        MultiIndex(levels=[[1, 2], [u'one', u'two']],
+        MultiIndex(levels=[[1, 2], ['one', 'two']],
                    labels=[[0, 0, 1, 1], [0, 1, 0, 1]],
-                   names=[u'baz', u'bar'])
+                   names=['baz', 'bar'])
+
+        See also
+        --------
+        rename : able to set new names without level
```

Comment:
See also should have a capital `A` (`See Also`) and should be placed before the examples. Also, the description should start with upper case and finish with a period.

Is `rename` the method of `Index`? For sphinx to create a link to it, it should be `Index.rename`. Just rename would be understood as `pandas.rename` which does not exist.